### ISSUE TITLE
Issue on page /Markdown/21_Einstieg_R.html

### ISSUE THREAD

**Issue by grotkopp** [quadriga-dk/Tabelle-Fallstudie-1#67]
Für den Fall, dass die Lernenden bereits in Kapitel 3 den csv Datensatz aus dem Repository heruntergeladen haben, und damit frisch und froh weiterarbeiten wollen, haben sie dort die Daten von 2013-2022. Ich würde empfehlen, darauf hinzuweisen, dass man das im Blick hat?

**Comment by JulianeSFh** [comment]
Ja, wir können darauf hinweisen, dass es sich um die gleiche Datei handelt, die bereits in Kapitel 3 runtergeladen wurde
